tests/platform_helper/test_command_pipeline.py

```diff
@@ -167,11 +166,14 @@ def test_pipeline_generate_with_empty_pipelines_yml_does_nothing(
 @patch("dbt_platform_helper.jinja2_tags.version", new=Mock(return_value="v0.1-TEST"))
 @patch("dbt_platform_helper.utils.aws.get_aws_session_or_abort")
 @patch("dbt_platform_helper.commands.pipeline.git_remote", return_value="uktrade/test-app-deploy")
-def test_pipeline_generate_overwrites_any_existing_config_files(
+def test_pipeline_generate_deletes_any_existing_config_files_and_writes_new_ones(
     git_remote, get_aws_session_or_abort, fakefs
 ):
     mock_codestar_connections_boto_client(get_aws_session_or_abort, ["test-app"])
     setup_fixtures(fakefs)
+    fakefs.add_real_file(
+        FIXTURES_DIR / "unnecessary_file.yml", False, "copilot/pipelines/unnecessary_file.yml"
+    )
```

Comment:
Can we create a file without needing the actual `unnecessary_file.yml` fixture?

Reply:
I'll look. Not massively familiar with the fakefs.

Reply:
`fakefs_create_file('filename', contents=""" some content """)` might be what you're after here, it's used in several tests